sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.institutions.controller.ts

```diff
@@ -157,29 +159,18 @@ export class ApplicationOfferingChangeRequestInstitutionsController extends Base
   > {
     const offeringChange =
       await this.applicationOfferingChangeRequestService.getSummaryByStatus(
-        locationId,
-        pagination,
         [
           ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
           ApplicationOfferingChangeRequestStatus.InProgressWithStudent,
         ],
+        pagination,
+        { locationId },
       );
     return {
-      results: offeringChange.results.map((eachOfferingChange) => {
-        const offering =
-          eachOfferingChange.application.currentAssessment.offering;
-        return {
-          id: eachOfferingChange.id,
-          applicationNumber: eachOfferingChange.application.applicationNumber,
-          applicationId: eachOfferingChange.application.id,
-          studyStartDate: offering.studyStartDate,
-          studyEndDate: offering.studyEndDate,
-          fullName: getUserFullName(
-            eachOfferingChange.application.student.user,
-          ),
-          status: eachOfferingChange.applicationOfferingChangeRequestStatus,
-        };
-      }),
+      results:
+        this.applicationOfferingChangeRequestControllerService.mapToInProgressApplicationOfferingChangesAPIOutDTOs(
```

Comment:
I see the for the sake AEST, some extra properties are returned institution, do we really need those properties in institution?

Reply:
If that is the case we can use the override to return the respective DTOs (AEST and Institution)

Reply:
As a suggestion I would say that instead of using controller service only for the mapping, we can have a common method like `getInProgressApplicationOfferingChangeRequests()` in controller service with override for AEST and Institution

Reply:
ok. Created a separate DTO. Thanks!